RELEASE.md

```diff
@@ -1,7 +1,7 @@
-<<<<<<< HEAD
 # Release 0.4.0
 - This release explicitly adds versions for dependencies, rather than leaving it up to the cellmlmanip and Jinja2 packages. Versions are semi-strict allowing for minor updates (which should not break compatibility).
 - This release contains a bug fix with regards to the stimulus sign, which was calculated incorrectly when the stimulus equation had been changed due to unit conversions.
+- The model type displayed in the header of generated files has been fixed and now displays correctlt (instead of always displaying normal)
```

Comment:
```suggestion
- The model type displayed in the header of generated files has been fixed and now displays correctly (instead of always displaying 'normal')
```